kylecornelissen/whats-cookin
# Pantry sort

#### What's this PR do?
- Allows user to check different ingredient boxes and look for recipes that contain that ingredient/those ingredients
- Adds and formats button that triggers the above function on click
#### Where should the reviewer start?
Main page
#### How should this be manually tested?
Click My Pantry and check 1 ingredient checkbox, click the What Can I Make? button. Now try with two ingredients
#### Any background context you want to provide?
Because we only have 50 recipes, a lot of the ingredient combos don't turn up any results. I've been using "brown sugar" and "butter" to test because I know they are both in a few recipes.
#### What are the relevant tickets?
#6
